build/ClientAnnotator.php

```diff
@@ -36,6 +50,17 @@ public function updateApiMethodAnnotations()
             '/^\* @method (\\\\Aws\\\\Result|\\\\GuzzleHttp\\\\Promise\\\\Promise) /'
         );
 
+        if ($this->reflectionInterface !== null) {
+            $interfaceUpdater = new ClassAnnotationUpdater(
+                $this->reflectionInterface,
+                $this->getMethodAnnotations(),
+                $this->getDefaultDocComment(),
+                '/^\* @method (\\\\Aws\\\\Result|\\\\GuzzleHttp\\\\Promise\\\\Promise) /'
+            );
+
+            return $interfaceUpdater->update() && $updater->update();
```

Comment:
question: can we just call `$interfaceUpdater->update()` here and fall through to the `$updater->update()` call?